SourceX/display.cpp

```diff
@@ -116,6 +121,12 @@ void CalculatePreferdWindowSize(int &width, int &height, bool useIntegerScaling)
 
 bool SpawnWindow(const char *lpWindowName)
 {
+
+#ifdef __vita__
+scePowerSetArmClockFrequency(444);
+#endif
+
+	SDL_SetHint(SDL_HINT_TOUCH_MOUSE_EVENTS, "0");
```

Comment:
```suggestion
#if !(SDL_VERSION_ATLEAST(2,0,6))
	SDL_SetHint(SDL_HINT_TOUCH_MOUSE_EVENTS, "0");
#endif
```

Reply:
Umm. Why `!` (not) ? 
Also, that hint first appeared in 2.0.10

Reply:
Oops, the `!` was a copy-paste error.

But the hint first appears in 2.0.6, https://github.com/spurious/SDL-mirror/blob/master/WhatsNew.txt#L329

What happens in 2.0.10 is that `SDL_HINT_ANDROID_SEPARATE_MOUSE_AND_TOUCH` is removed in favor of the then-existing `SDL_HINT_TOUCH_MOUSE_EVENTS`.